management-service/appcontext/appcontext.go

```diff
@@ -30,36 +32,47 @@ func NewAppContext(db *gorm.DB, authorizer *mw.Authorizer, cfg *config.Config) (
 	}
 
 	// Init Services
+	log.Println("Initializing message queue publisher...")
 	messageQueueService, err := messagequeue.NewMessageQueueService(*cfg.MessageQueueConfig)
 	if err != nil {
 		return nil, err
 	}
 
+	log.Println("Initializing segmenter service...")
 	segmenterSvc, err := services.NewSegmenterService(&allServices, cfg.SegmenterConfig, db)
 	if err != nil {
 		return nil, err
 	}
 
+	log.Println("Initializing validation service...")
 	validationService, err := services.NewValidationService(cfg.ValidationConfig)
 	if err != nil {
 		return nil, err
 	}
 
+	log.Println("Initializing experiment history service...")
 	experimentHistorySvc := services.NewExperimentHistoryService(db)
+	log.Println("Initializing experiment service...")
 	experimentSvc := services.NewExperimentService(&allServices, db)
+	log.Println("Initializing project settings service...")
 	projectSettingsSvc := services.NewProjectSettingsService(&allServices, db)
 
+	log.Println("Initializing segment history service...")
 	segmentHistorySvc := services.NewSegmentHistoryService(db)
+	log.Println("Initializing segment service...")
 	segmentSvc := services.NewSegmentService(&allServices, db)
 
+	log.Println("Initializing treatment history service...")
 	treatmentHistorySvc := services.NewTreatmentHistoryService(db)
+	log.Println("Initializing treatment service...")
```

Comment:
These seem like Debug logs. It would be enough to just have error logs if the initialization fails. Alternatively, if we want to keep these logs, we should log them at the appropriate level.